api/server/server.go

```diff
@@ -138,12 +138,7 @@ func (s *Server) makeHTTPHandler(handler httputils.APIFunc) http.HandlerFunc {
 		}
 
 		if err := handlerFunc(ctx, w, r, vars); err != nil {
-			statusCode := httputils.GetHTTPErrorStatusCode(err)
-			errFormat := "%v"
-			if statusCode == http.StatusInternalServerError {
-				errFormat = "%+v"
-			}
-			logrus.Errorf("Handler for %s %s returned error: "+errFormat, r.Method, r.URL.Path, err)
+			logrus.Errorf("Handler for %s %s returned error: %v", r.Method, r.URL.Path, err)
```

Comment:
It'd be nice to only log for 50x errors.
Could do debug for everything else if we really need it, but error messages in the logs for, e.g. , 404's is yuck.